src/ad-bidders/prebid/intent-iq/intent-iq.ts

```diff
@@ -130,15 +130,15 @@ export class IntentIQ {
 		);
 	}
 
-	private getPPID(data: IntentIqData): string | undefined {
+	getPpid(data: IntentIqData): string | undefined {
 		try {
 			return this.extractId(data);
 		} catch (error) {
 			utils.warner(logGroup, 'error setting ppid', error);
 		}
 	}
 
-	private setPpid(ppid: string): void {
+	setPpid(ppid: string): void {
 		targetingService.set('intent_iq_ppid', ppid);
```

Comment:
```suggestion
		targetingService.set('intent_iq_ppid', ppid, 'intent_iq');
```
otherwise it will send keyval to GAM